Mathlib/Order/Basic.lean

```diff
@@ -782,6 +782,17 @@ instance IsRefl.compl (r) [IsRefl α r] : IsIrrefl α rᶜ :=
   ⟨fun a ↦ not_not_intro (refl a)⟩
 #align is_refl.compl IsRefl.compl
 
+instance DecidableRel.compl (r : α → α → Prop) [i : DecidableRel r] : DecidableRel rᶜ :=
```

Comment:
Is this not already inferred?

Reply:
Oh. I guess it is. I was struggling to get things with complements to work, and it seemed like this was necessary ... but I guess it isn't. Will cut it out.